packages/composite-checkout-wpcom/package.json

```diff
@@ -39,13 +39,15 @@
 		"@automattic/composite-checkout": "file:../composite-checkout",
 		"@emotion/core": "10.0.22",
 		"@emotion/styled": "10.0.23",
-		"@wordpress/data": "^4.9.2",
 		"debug": "4.1.1",
 		"emotion-theming": "10.0.19",
 		"i18n-calypso": "file:../i18n-calypso",
 		"prop-types": "^15.7.2",
-		"react": "^16.8.6",
 		"react-stripe-elements": "^5.1.0"
 	},
+	"peerDependencies": {
+		"@wordpress/data": "^4",
```

Comment:
`composite-checkout-wpcom` depends on `composite-checkout` which uses data hooks and depends on `wp-data@^4.6`. So, this `^4` peer dependency can never actually resolve to a version older than `^4.6`. But that's just very subtle nitpick, doesn't matter.